team-validator.js

```diff
@@ -214,6 +215,14 @@ class Validator {
 				return [`"${set.ability}" is an invalid ability.`];
 			}
 		}
+		if (tools.getNature(set.nature).id && !tools.getNature(set.nature).name.exists) {
```

Comment:
You are aware `set.nature.name.exists` will always be undefined, right? Since `set.nature.name` is expected to be "Modest"/"Timid"/"Jolly"/etc. and doesn't have an `exists` property.